### PR TITLE
Sort services in columns.

### PR DESCRIPTION
**Pull request by kaj**
The main selected services list is two columns, not a bunch of rows, so "Annual ..." comes below "Administrate", not beside.  Also a bunch of cleanup of the main layout of the services panel (less calc, fewer properties, let the flex handling in the browser do the math).

https://trello.com/c/n0GW8mHA

### CODE REVIEW

File: kpm/kpm-frontend/src/panes/services.scss [hunk 1]
```diff
@@ -1,35 +1,18 @@
...
-      flex-flow: row wrap;
-      align-items: flex-start;
-      gap: $rowGap $colGap;
-
-      & > li {
-        $colCount: 2;
-        width: calc((100% - $colGap) / $colCount);
-        margin-bottom: 0;
-      }
+      column-width: 17em;
```
Comment (jhsware) [inline kpm/kpm-frontend/src/panes/services.scss:14]: The value feels picked at random. Would be great with a comment OR using a calculation from a variable that is used somewhere related.

File: kpm/kpm-frontend/src/panes/services.scss [hunk 2]
```diff
@@ -1,35 +1,18 @@
...
-
-  display: flex;
-  flex-flow: row wrap;
   align-items: flex-start;
-  gap: $rowGap $colGap;
 
   .kpm-col {
-    width: $colWidth;
-    flex-grow: 0;
+    flex: 13em 1;
```
Comment (jhsware) [inline kpm/kpm-frontend/src/panes/services.scss:7]: This value also feels random. See other note.

**Review by jhsware — CHANGES_REQUESTED** [review]
The result looks good, but I have some comments regarding understandability.

File: kpm/kpm-frontend/src/panes/services.scss (hunk 1 above)
Comment (kaj) [reply kpm/kpm-frontend/src/panes/services.scss:14]: It is picked by trying different values and selecting one that looks good.  Not sure what to write in a comment about that, other than something like "this seems fine", which don't add much value ...

File: kpm/kpm-frontend/src/panes/services.scss (hunk 2 above)
Comment (kaj) [reply kpm/kpm-frontend/src/panes/services.scss:7]: Same.  :-)

**Review by jhsware — APPROVED** [review]
LGTM

**Review by jhsware — APPROVED** [review]
LGTM

**Pull request merged**